src/utils/parse-markdown-in-locale.ts

```diff
@@ -0,0 +1,44 @@
+import get from 'lodash/get';
+import set from 'lodash/set';
+import { TALLLanguages } from '~/locale';
+import { MDToHTMLString } from './MDToHTMLString';
+
+/**
+ * This is a list of keys which contain markdown syntax
+ */
+const MARKDOWN_KEYS = [
+  'regionaal_index.belangrijk_bericht',
+  'veiligheidsregio_index.selecteer_toelichting',
+  'veiligheidsregio_index.belangrijk_bericht',
+  'over_risiconiveaus.toelichting',
+  'over_beschrijving.text',
+  'over_veelgestelde_vragen.vragen[].antwoord',
+  'verantwoording.cijfers[].verantwoording',
+];
+
+export function parseMarkdownInLocale(text: TALLLanguages) {
+  const textClone = JSON.parse(JSON.stringify(text)) as TALLLanguages;
+
+  MARKDOWN_KEYS.forEach((key) => replaceWithMarkdown(textClone, key));
+
+  return textClone;
+}
+
+function replaceWithMarkdown(object: any, path: string) {
```

Comment:
Any can be stricter if you expect an object
```suggestion
function replaceWithMarkdown(object: Record<string, unknown>, path: string) {
```